src/OI.cpp

```diff
@@ -43,7 +47,11 @@ OI::OI()
 
   SmartDashboard::PutData("TankDrive", new TankDrive());
   SmartDashboard::PutData("Autonomous Command", new AutonomousCommand());
-
+  SmartDashboard::PutData("FeederOn", new ConveyorControl(1.0, 0.0));
+  SmartDashboard::PutData("MoverOn", new ConveyorControl(0.0, 1.0));
+  SmartDashboard::PutData("IntakeOn", new SetIntake(1.0));
+  SmartDashboard::PutData("ShooterOn", new SetShooter(1.0));
+  SmartDashboard::PutData("TurretOn", new SetTurret(1.0));
```

Comment:
Thanks @Diego-Andrade. 

Walking through the process ... 

1. Press Start. The command starts running on this Talon. The button changes to Cancel.
2. Press Cancel.
  a. The current command is interrupted and exits.
  b. The default command is instantiated and restarted.

But wait ... most of these subsystems have no default command. Due to the implementation of the Talon, the previous setting keeps running. 

I think we should have taken your (@Diego-Andrade) early suggestion to write default commands for all of these systems. 

Do we have default commands for each of these subsystems that disables the talons (ie set the motor to 0?).